src/platform/plugins/shared/controls/public/controls/data_controls/range_slider/editor_state_manager.ts

```diff
@@ -18,7 +18,7 @@ export const editorComparators: StateComparators<EditorState> = {
 };
 
 const defaultEditorState = {
-  step: 1,
+  step: undefined,
```

Comment:
Range slider was triggering unsaved changes for legacy dashboards - the default should be undefined.